configs/quality/scripts_lifecycle_registry.json

```
@@ -1369,6 +1369,12 @@
       "owner": "@bioetl-platform",
       "review_by": "2026-11-17"
     },
+    "scripts/ops/observability/grafana/action_target_routes.py": {
```

Biome (2.5.8)

[error] The key scripts/ops/observability/grafana/action_target_routes.py was already declared.

(lint/suspicious/noDuplicateObjectKeys)